packages/effect/src/DateTime.ts

```diff
@@ -444,14 +444,12 @@ export const unsafeNow: LazyArg<Utc> = Internal.unsafeNow
  * @category time zones
  * @example
  * ```ts
- * import { DateTime, Effect } from "effect"
+ * import { DateTime } from "effect"
  *
- * Effect.gen(function* () {
- *   const now = yield* DateTime.now
+ * const now = yield* DateTime.unsafeMakeZoned({ year: 2024 }, { timeZone: "Europe/London" })
```

Comment:
`unsafeMakeZoned` isn't an Effect, and examples need to be runnable (otherwise CI won't pass)